src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,83 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+type Props = {
+  searchMovieTitle: (title:string) => void;
+  IsMovieFounded: boolean;
+  foundedMovie: Movie | null;
+  addToList: () => void;
+};
+
+export const FindMovie: React.FC<Props> = (
+  {
+    searchMovieTitle,
+    IsMovieFounded,
+    foundedMovie,
+    addToList,
+  },
+) => {
+  const [inputQuery, setInputQuery] = useState('');
+
+  const handleInput = (e:React.ChangeEvent<HTMLInputElement>) => {
+    const searchedTitle = e.target.value;
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+    setInputQuery(searchedTitle);
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className="input is-danger"
+              onChange={e => handleInput(e)}
+            />
+          </div>
+          {!IsMovieFounded && (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
```

Comment:
Use template strings for it

Reply:
how🙂

Reply:
```suggestion
              {`Can't find a movie with such a title`}
```

Reply:
strings must be singlequotes

Reply:
lint doesn't allow to do this

Reply:
should i disable lint?

Reply:
and by the way, i didn't write this line, it was there

Reply:
k, will pass on that one))